external/codecs/Make.defs

```diff
@@ -15,7 +15,7 @@
 # language governing permissions and limitations under the License.
 #
 ###########################################################################
-# apps/netutils/ftpc/Make.defs
+# apps/netutils/codecs/Make.defs
 # Adds selected applications to apps/ build
```

Comment:
Adds selected applications to apps/ build
==>
Adds selected applications to external/ build